.vsts-ci/mac.yml

```diff
@@ -9,11 +9,13 @@ variables:
 resources:
   - repo: self
     clean: true
-phases:
-- phase: Build
 
-  queue:
-    name: Hosted macOS
+jobs:
+- job: Build
+
```

Comment:
Add `displayName:` for job?

Reply:
I'm following the convention of the other ymls which has displayname for steps, but not the job